### ISSUE TITLE
CvdMatch property in CardResponse is always null

### ISSUE THREAD

**Issue by Sakurina** [bambora-na/bambora-na-dotnet#34]
The [CardResponse class][cr] has an optional integer property called *CvdMatch*, based on the JSON property *cvd_match*. Unfortunately, this JSON property is never returned by the API, so it is always null.

Would it be possible to remove this property entirely to avoid having people write code mistakingly using this property instead of *CvdResult*?

[cr]: https://github.com/bambora-na/bambora-na-dotnet/blob/master/BamboraSDK/Responses/CardResponse.cs

**Comment by svenresch** [comment]
Agreed! Thx for the feedback. We will create a ticket to get this fixed.
